astropy/io/votable/tree.py

```diff
@@ -2420,7 +2420,10 @@ def __init__(
         warn_unknown_attrs("TABLE", extra.keys(), config, pos)
 
     def __repr__(self):
-        return repr(self.to_table())
+        s = repr(self.to_table())
+        if s.startswith("<Table"):
+            s = "<VO" + s[1:]
```

Comment:
To guard against something else breaking unexpectedly because this is old code and I don't understand the whole stack in this subpackage. So I chose to be very explicit on when to overwrite the repr. Did this if break stuff for you?

Reply:
No break, I'm just about to do some more similar things in pyvo and spotted this as something odd

Reply:
If you are sure you don't need the if, you can remove it. I am just not 100% sure and don't have time to investigate. Thanks!